src/applications/facility-locator/components/MessagePromptDiv.jsx

```diff
@@ -6,9 +6,11 @@ const MessagePromptDiv = ({ id, message, waitBeforeShow = 0 }) => {
 
   useEffect(
     () => {
-      setTimeout(() => {
+      const timeoutFn = setTimeout(() => {
         setIsShown(true);
       }, waitBeforeShow);
+
+      return () => clearTimeout(timeoutFn);
```

Comment:
Added this as cleanup functions are necessary in `useEffects` for setting timeouts and there was a warning about it in the console where this appeared.